src/lib.rs

```diff
@@ -348,6 +348,304 @@ languages_table!(
     (Zu, "zu", "zul", "zul", "Zulu", "Niger–Congo"),
 );
 
+#[cfg(feature = "iterable")]
+pub mod iter {
+    use super::LanguageCode;
+
+    #[derive(Debug, Default, Clone)]
```

Comment:
Can you add documentation to this and the other iterators? Something as simple as

```rust
/// An iterator over the values of a `LanguageCode`.
```

Should do fine.